src/components/pages/devices/flyouts/deviceDetails/propertyGrid/propertyGridBody.js

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
You don't need to do this in the JS. You can actually do it all in the CSS. Take a look at [this example](https://jsfiddle.net/1n0axa3y/110/) I put together. In this example, I can add or remove the div with the `scrollable` class name and the grid will convert from a scrollable to a non-scrollable grid automatically with just CSS and minimal markup changes. It is also variable easy to read the JSX and understand what each component is doing.